core/src/main/scala/org/http4s/EntityDecoder.scala

```diff
@@ -177,4 +177,11 @@ trait EntityDecoderInstances {
 
   implicit def multipart: EntityDecoder[Multipart] =
     MultipartDecoder.decoder
+
+  /** An entity decoder that ignores the content and returns unit. */
+  implicit val void: EntityDecoder[Unit] = new EntityDecoder {
+    val consumes: Set[MediaRange] = Set.empty
```

Comment:
I think this could be ``MediaRange.`*.*` `` -- we can consume anything with this.